ocs_ci/deployment/baremetal.py

```diff
@@ -537,6 +537,39 @@ def create_pxe_files(self, ocp_version, role):
                 t_file.writelines(default_pxe_file)
             return temp_file.name
 
+        def set_pxe_boot_and_reboot(self, machine):
+            """
+            Ipmi Set Pxe boot and Restart the machine
+
+            Args:
+                machine (str): Machine Name
+
+            Returns:
```

Comment:
Remove the Returns section since this function doesn't return anything

Reply:
Done